portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/contentsecuritypolicy/ContentSecurityPolicy.testcase

```diff
@@ -0,0 +1,80 @@
+@component-name = "portal-security"
+definition {
+
+	property ci.retries.disabled = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Content Security Policy";
```

Comment:
Need to add "Content Security Policy" under testray.team.security.component.names=\